docs/config/build-options.md

```diff
@@ -244,3 +244,15 @@ There are cases that file system watching does not work with WSL2.
 See [`server.watch`](./server-options.md#server-watch) for more details.
 
 :::
+
+
+## build.disabled
+
+- **Experimental**
+- **Type:** `boolean | string`
+- **Default:** `disabled`
+
+Use esbuild to optimize dependencies during build time. If enabled, it removes one of the most significant differences between dev and prod present in v2. [`@rollup/plugin-commonjs`](https://github.com/rollup/plugins/tree/master/packages/commonjs) is no longer needed in this case since esbuild converts CJS-only dependencies to ESM.
+
+If you want to try this build strategy, you can use `optimizeDeps.disabled: false` (the default in v3 is `disabled: 'build'`). `@rollup/plugin-commonjs`
+can be removed by passing `build.commonjsOptions: { include: [] }`.
```

Comment:
i'm confused by this line. it makes it sound like the setting name is `optimizeDeps.disabled`, so why is it then called `build.disabled` here?

also, don't need to manually wrap the docs
```suggestion
If you want to try this build strategy, you can use `optimizeDeps.disabled: false` (the default in v3 is `disabled: 'build'`). `@rollup/plugin-commonjs` can be removed by passing `build.commonjsOptions: { include: [] }`.
```